pysqldb3/pysqldb3.py

```diff
@@ -2172,45 +2178,126 @@ def gpkg_to_table(self, dbo = None, path=None, schema=None, table =None, gpkg_na
             gpkg_name = os.path.basename(filename)
             path = os.path.dirname(filename)
 
+        gpkg = Geopackage(path=path, gpkg_name=gpkg_name, gpkg_tbl = gpkg_tbl)
+
         if not table:
             table = gpkg_tbl.replace('.gpkg', '').lower() # if the gpkg_table is left blank, we will populate the name using read_gpkg
         
-        table = table.lower()
+        if bulk_upload == True:
+            # bulk upload
+            output_table_names = gpkg.read_gpkg_bulk_upload(dbo = dbo, schema = schema, gpkg_encoding = gpkg_encoding, srid = srid,
+                            gdal_data_loc=gdal_data_loc, precision = precision, private = private, print_cmd = print_cmd)
+            
+            for table in output_table_names.values():
 
-        gpkg = Geopackage(path=path, gpkg_name=gpkg_name, gpkg_tbl = gpkg_tbl)
+                if self.type == "MS":
 
-        gpkg.read_gpkg(dbo = dbo, schema = schema, table = table, gpkg_encoding = gpkg_encoding, srid = srid,
-                       gdal_data_loc=gdal_data_loc, precision = precision, private = private, print_cmd = print_cmd)
+            # rename geom columns if necessary (problem only identified in MS)
+                    try:
+                        geom_output = self.dfquery(f""" SELECT COLUMN_NAME 
+                                                    FROM information_schema.COLUMNS 
+                                                    WHERE lower(TABLE_NAME)=lower('{table}')
+                                                    and table_schema = '{schema}'
+                                                    AND DATA_TYPE = 'geometry' """).values[0][0]
+
+                        if geom_output != 'geom':      
+                            self.query(f"EXEC sp_rename '{schema}.{table}.{geom_output}', 'geom', 'COLUMN'")
+                    except:
+                        pass
+                else:
+                    # Postgres
+                    try:
+                        geom_output = self.dfquery(f"""SELECT COLUMN_NAME 
+                                        FROM information_schema.COLUMNS 
+                                        WHERE data_type = 'USER-DEFINED'
+                                        AND lower(TABLE_NAME)=lower('{table}')
+                                        AND table_schema = '{schema}'
+                                """).values[0][0]
+                    
+                        if geom_output != 'geom':
+                            self.query(f"alter table {schema}.{table} rename column {geom_output} to geom")
```

Comment:
all of these internally used query calls should include the `internal=True` flag  so they arent stored with the user generated queries, thats something that needs to be cleaned up generally in the code since that is a new flag